.github/workflows/notes.md

```diff
@@ -18,12 +18,18 @@ ${NVIM_VERSION}
 2. Run the MSI
 3. Search and run `nvim-qt.exe` or run `nvim.exe` on your CLI of choice.
 
-### macOS
+### macOS 10.11+
 
 1. Download **nvim-macos.tar.gz**
 2. Extract: `tar xzvf nvim-macos.tar.gz`
 3. Run `./nvim-osx64/bin/nvim`
 
+### macOS 11+ (Universal)
```

Comment:
do we need both? or can we have only the universal build?

Reply:
Big +1 for just providing a single universal build.

Reply:
Having a single universal build is tricky. Ideally you'll want to set different `MACOSX_DEPLOYMENT_TARGET`/`CMAKE_OSX_DEPLOYMENT_TARGET`s for each arch slice (`10.11` for `x86_64`, `11` for `arm64`), and `cmake` doesn't make it easy to set arch-dependent depoloyment targets. Setting only one means you end up with a binary with restricted portability (deployment target too high) or with a possibly broken `arm64` slice (deployment target too low).

One way to do this is to build for each arch separately and then use `lipo` to bundle them all up.

If I can get this approach for a universal build working then this alternative approach (build separately + bundle together with `lipo`) should be relatively straightforward.

Reply:
> If I can get this approach for a universal build working then this alternative approach (build separately + bundle together with `lipo`) should be relatively straightforward.

I lied; sorry. Under a default build and in the absence of the ability to run ARM64 binaries on an X64 machine, this is pretty much impossible. This is because we pre-compile Lua sources into byte-code, which requires a `lua[jit]` compiler that runs on the host machine.

Reply:
Works for me. Though some initial testing suggests that we need not make that tradeoff.

This hinges, however, on whether the byte-code compiled with an x64 luajit and embedded into the (universal) nvim binary will work when executed on an arm64 machine. Would you know? I was actually already contemplating on tagging you here to ask this, so it's convenient you've dropped by.

Reply:
According to https://luajit.org/extensions.html#string_dump:

>The generated bytecode is portable and can be loaded on any architecture that LuaJIT supports, independent of word size or endianess. However the bytecode compatibility versions must match. Bytecode stays compatible for dot releases (x.y.0 → x.y.1), but may change with major or minor releases (2.0 → 2.1) or between any beta release.

So it sounds like it should work.